src/index.tsx

```diff
@@ -50,7 +54,8 @@ const ReactChatbotInternal: FC<Props> = ({
   emptyStateDisplay,
   isInitiallyOpen,
   zIndex,
-  enableStreaming = true
+  enableStreaming = true,
+  language = "eng"
```

Comment:
Do we need to set defaults here? They're already being set in `useChat` so it seems like these might be redundant. Maybe they can be removed?

Reply:
Another option is to extract these default values out into a third file, that both this code and the hook consume. That way there's a single source of truth.